### DISCUSSION TITLE
Anchors & links for Heading 1

### DISCUSSION THREAD

**Discussion by AnnaSeredina** [requarks/wiki#6317] [Error / Bug Report]
The anchors and links for Heading 1 are not working. The anchor icon is missing.
All updates are installed. 
What could be the problem? How to fix it? 

OS: Docker Container (Linux)
Wiki.js version: 2.5.297
Database engine: PostgreSQL 11.8

<img width="568" alt="Знімок екрана 2023-04-03 о 14 19 51" src="https://user-images.githubusercontent.com/94693702/229497182-aecd5821-2840-465d-9a97-ff0ef30d63bc.png">
<img width="1434" alt="Знімок екрана 2023-04-03 о 14 25 47" src="https://user-images.githubusercontent.com/94693702/229497189-a8ca19b4-712e-4424-a6c7-d7737d89dadb.png">

**Comment by NGPixel** [comment]
You need to mouse-over the header for them to appear.

**Comment by alekseydmytrenko** [comment]
I have the same issue and actually not just with Heading 1 type of titles. When you click on link I can see an error in a console `TypeError: Target must be a Number/Selector/HTMLElement/VueComponent, received undefined instead.` Any ideas? Thanks

**Reply by FedeGarcia00** [reply]
same problem here, were you able to fix it?
